docs/earthpy-data-subsets.rst

```diff
@@ -0,0 +1,83 @@
+=====================
+EarthPy Data Subsets
+=====================
+
+EarthPy comes with the ability to download example datasets using
+``earthpy.data.get_data()``. These datasets are used in example documentation
+on how to use EarthPy, as well as in the EarthPy related lessons on
+earthdatascience.org. There are many keys that can be used to get data
+this way, all of which can be found below.
+
+Data Keys
+=========
+
+co-flood-extras
+---------------
+
+Contains two additional csv files for the Colorado Flood exercise on earthdatascience.org.
+Includes precipitation and `temperature <https://figshare.com/articles/Week_2_earth_analytics/4502138>`_
```

Comment:
The link for precipitation is: https://figshare.com/articles/CO_Flood_2013_-_Classroom_Lab/4295360
(It is the precipboulderaugoct2013.csv.)